lib/api/Cargo.toml

```diff
@@ -162,3 +162,4 @@ features = [
     "wasmer-artifact-create",
     "wasmer-artifact-load",
 ]
+rustc-args = ["--cfg", "docsrs"]
```

Comment:
Do we want this?

Will this make the rebuild situation even worse due to RUSTFLAGS mismatch?

Reply:
It should be fine because the `rustc-args` flag is only used by docs.rs's when it's building docs. It's hard to see because of how GitHub's UI collapses things, but `rustc-args` is under the `[package.metadata.docs.rs]` table.

Reply:
Yeah I noticed this after, I initially thought it was in the main build section.